src/frequenz/sdk/actor/power_distributing/_component_pool_status_tracker.py

```diff
@@ -0,0 +1,163 @@
+# License: MIT
+# Copyright © 2022 Frequenz Energy-as-a-Service GmbH
```

Comment:
```suggestion
# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
```

Reply:
This is actually a renaming, not a new file: https://github.com/frequenz-floss/frequenz-sdk-python/pull/779/commits/278b0b92475dc3d42566560a5d4493095f7bdd26#diff-40a429480bdbbab4e414638f059fa7202095acd519c72353f9ad2a762e75cad7